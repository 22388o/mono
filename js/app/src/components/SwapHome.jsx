import React from 'react'

import { Grid } from '@mui/material'

import styles from '../styles/SwapHome.module.css'
import { SwapCreate } from './SwapCreate/SwapCreate'
import { WalletInfoContainer } from './WalletInfoContainer'

/**
 * Swap Page Component
 */
export const SwapHome = () => {
  return (
    <Grid container direction='column' sx={{backgroundColor:'#101010'}}>
      <Grid direction='row' container className={styles['page-header']}>
        
      </Grid>
      <Grid item container justifyContent='flex-end' style={{ padding: '1em 2em' }}>

      </Grid>
      <Grid item container direction='row'>
        <Grid item container xs={10} sx={{ transition: 'all 0.5s ease-in-out' }}>
          <SwapCreate />
        </Grid>
      </Grid>

      <WalletInfoContainer show={true} setIsMinimized={() => {}} />
    </Grid>
  )
}
