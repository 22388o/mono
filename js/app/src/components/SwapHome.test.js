import { create, act } from 'react-test-renderer'
import { SwapHome } from './SwapHome';

global.fetch = jest.fn(() =>
  Promise.resolve({
    json: () => Promise.resolve({ data: { amount: 12 } }),
  })
);

describe('SwapHome test', () => {
  it('should work', () => {
    let tree

    act(() => {
      tree = create(
        <SwapHome />
      )
    })
    expect(tree).toMatchSnapshot()

  })
  afterAll(() => jest.resetModules())
})
