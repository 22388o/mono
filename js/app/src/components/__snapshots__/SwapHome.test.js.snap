// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwapHome test should call handler when input 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-uz3bcw-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    />
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-hoc6b0-MuiGrid-root"
      style="padding: 1em 2em;"
    />
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-1f064cs-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10 css-1dsmj4t-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="MuiStack-root css-163ljxj-MuiStack-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container css-1o9533o-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-z58hwb-MuiGrid-root"
                style="display: flex; align-items: center;"
              >
                <h3>
                  Swap
                </h3>
              </div>
            </div>
            <div
              class="MuiGrid-root css-vj1n65-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1tj2tdk-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-7 css-1smoqvn-MuiGrid-root"
                >
                  <input
                    class="undefined qty-input"
                    placeholder="1000"
                    type="number"
                    value="12"
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-14m1l2r-MuiGrid-root"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium undefined coin-select css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <img
                      src="https://github.com/dapphub/trustwallet-assets/blob/master/blockchains/bitcoin/info/logo.png?raw=true"
                    />
                    BTC
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
                style="border-color: #202020; margin: 0.5em -1em 0px -1em;"
              />
              <button
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium undefined exchange css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="SettingsEthernetIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7.77 6.76 6.23 5.48.82 12l5.41 6.52 1.54-1.28L3.42 12l4.35-5.24zM7 13h2v-2H7v2zm10-2h-2v2h2v-2zm-6 2h2v-2h-2v2zm6.77-7.52-1.54 1.28L20.58 12l-4.35 5.24 1.54 1.28L23.18 12l-5.41-6.52z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <div
                class="MuiGrid-root MuiGrid-container mt-m1 mb-0 css-1tj2tdk-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-7 css-1smoqvn-MuiGrid-root"
                >
                  <input
                    class="undefined qty-input"
                    placeholder="1000"
                    type="number"
                    value="123"
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-14m1l2r-MuiGrid-root"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium undefined coin-select css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <img
                      src="https://github.com/dapphub/trustwallet-assets/blob/master/blockchains/ethereum/info/logo.png?raw=true"
                    />
                    ETH
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root css-e54n9e-MuiGrid-root"
            >
              <p>
                Loading
              </p>
              <button
                circular="true"
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium gradient-btn w-100 h-3 css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                secondary="true"
                tabindex="0"
                type="button"
              >
                Swap
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      style="display: block;"
    >
      <div
        class="MuiStack-root css-1rzuxdl-MuiStack-root"
      >
        <div
          class="MuiStack-root css-m69qwo-MuiStack-root"
        >
          <div
            class="MuiStack-root css-313u06-MuiStack-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 undefined undefined css-ahj2mt-MuiTypography-root"
            >
              Activity
            </p>
          </div>
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="KeyboardDoubleArrowRightIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M6.41 6 5 7.41 9.58 12 5 16.59 6.41 18l6-6z"
              />
              <path
                d="m13 6-1.41 1.41L16.17 12l-4.58 4.59L13 18l6-6z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-1603n5b-MuiTypography-root"
          >
            No Activities
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`SwapHome test should call handler when input 2`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-uz3bcw-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    />
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-hoc6b0-MuiGrid-root"
      style="padding: 1em 2em;"
    />
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-1f064cs-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10 css-1dsmj4t-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="MuiStack-root css-163ljxj-MuiStack-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container css-1o9533o-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-z58hwb-MuiGrid-root"
                style="display: flex; align-items: center;"
              >
                <h3>
                  Swap
                </h3>
              </div>
            </div>
            <div
              class="MuiGrid-root css-vj1n65-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1tj2tdk-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-7 css-1smoqvn-MuiGrid-root"
                >
                  <input
                    class="undefined qty-input"
                    placeholder="1000"
                    type="number"
                    value=""
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-14m1l2r-MuiGrid-root"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium undefined coin-select css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <img
                      src="https://github.com/dapphub/trustwallet-assets/blob/master/blockchains/bitcoin/info/logo.png?raw=true"
                    />
                    BTC
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
                style="border-color: #202020; margin: 0.5em -1em 0px -1em;"
              />
              <button
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium undefined exchange css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="SettingsEthernetIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7.77 6.76 6.23 5.48.82 12l5.41 6.52 1.54-1.28L3.42 12l4.35-5.24zM7 13h2v-2H7v2zm10-2h-2v2h2v-2zm-6 2h2v-2h-2v2zm6.77-7.52-1.54 1.28L20.58 12l-4.35 5.24 1.54 1.28L23.18 12l-5.41-6.52z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <div
                class="MuiGrid-root MuiGrid-container mt-m1 mb-0 css-1tj2tdk-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-7 css-1smoqvn-MuiGrid-root"
                >
                  <input
                    class="undefined qty-input"
                    placeholder="1000"
                    type="number"
                    value=""
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-14m1l2r-MuiGrid-root"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium undefined coin-select css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <img
                      src="https://github.com/dapphub/trustwallet-assets/blob/master/blockchains/ethereum/info/logo.png?raw=true"
                    />
                    ETH
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root css-e54n9e-MuiGrid-root"
            >
              <button
                circular="true"
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium Mui-disabled MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium w-100 h-3 gradient-btn-disabled css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                disabled=""
                secondary="true"
                tabindex="-1"
                type="button"
              >
                Enter Amounts to Swap
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      style="display: block;"
    >
      <div
        class="MuiStack-root css-1rzuxdl-MuiStack-root"
      >
        <div
          class="MuiStack-root css-m69qwo-MuiStack-root"
        >
          <div
            class="MuiStack-root css-313u06-MuiStack-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 undefined undefined css-ahj2mt-MuiTypography-root"
            >
              Activity
            </p>
          </div>
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="KeyboardDoubleArrowRightIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M6.41 6 5 7.41 9.58 12 5 16.59 6.41 18l6-6z"
              />
              <path
                d="m13 6-1.41 1.41L16.17 12l-4.58 4.59L13 18l6-6z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-1603n5b-MuiTypography-root"
          >
            No Activities
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`SwapHome test should work 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-uz3bcw-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    />
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-hoc6b0-MuiGrid-root"
      style="padding: 1em 2em;"
    />
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-1f064cs-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10 css-1dsmj4t-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="MuiStack-root css-163ljxj-MuiStack-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container css-1o9533o-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-z58hwb-MuiGrid-root"
                style="display: flex; align-items: center;"
              >
                <h3>
                  Swap
                </h3>
              </div>
            </div>
            <div
              class="MuiGrid-root css-vj1n65-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1tj2tdk-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-7 css-1smoqvn-MuiGrid-root"
                >
                  <input
                    class="undefined qty-input"
                    placeholder="1000"
                    type="number"
                    value=""
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-14m1l2r-MuiGrid-root"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium undefined coin-select css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <img
                      src="https://github.com/dapphub/trustwallet-assets/blob/master/blockchains/bitcoin/info/logo.png?raw=true"
                    />
                    BTC
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
                style="border-color: #202020; margin: 0.5em -1em 0px -1em;"
              />
              <button
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium undefined exchange css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="SettingsEthernetIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7.77 6.76 6.23 5.48.82 12l5.41 6.52 1.54-1.28L3.42 12l4.35-5.24zM7 13h2v-2H7v2zm10-2h-2v2h2v-2zm-6 2h2v-2h-2v2zm6.77-7.52-1.54 1.28L20.58 12l-4.35 5.24 1.54 1.28L23.18 12l-5.41-6.52z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <div
                class="MuiGrid-root MuiGrid-container mt-m1 mb-0 css-1tj2tdk-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-7 css-1smoqvn-MuiGrid-root"
                >
                  <input
                    class="undefined qty-input"
                    placeholder="1000"
                    type="number"
                    value=""
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-14m1l2r-MuiGrid-root"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium undefined coin-select css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <img
                      src="https://github.com/dapphub/trustwallet-assets/blob/master/blockchains/ethereum/info/logo.png?raw=true"
                    />
                    ETH
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root css-e54n9e-MuiGrid-root"
            >
              <button
                circular="true"
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium Mui-disabled MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium w-100 h-3 gradient-btn-disabled css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                disabled=""
                secondary="true"
                tabindex="-1"
                type="button"
              >
                Enter Amounts to Swap
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      style="display: block;"
    >
      <div
        class="MuiStack-root css-1rzuxdl-MuiStack-root"
      >
        <div
          class="MuiStack-root css-m69qwo-MuiStack-root"
        >
          <div
            class="MuiStack-root css-313u06-MuiStack-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 undefined undefined css-ahj2mt-MuiTypography-root"
            >
              Activity
            </p>
          </div>
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="KeyboardDoubleArrowRightIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M6.41 6 5 7.41 9.58 12 5 16.59 6.41 18l6-6z"
              />
              <path
                d="m13 6-1.41 1.41L16.17 12l-4.58 4.59L13 18l6-6z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-1603n5b-MuiTypography-root"
          >
            No Activities
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`SwapHome test should work 2`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-uz3bcw-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    />
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-hoc6b0-MuiGrid-root"
      style="padding: 1em 2em;"
    />
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-item css-1f064cs-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-10 css-1dsmj4t-MuiGrid-root"
      >
        <div
          class="MuiBox-root css-0"
        >
          <div
            class="MuiStack-root css-163ljxj-MuiStack-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container css-1o9533o-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-z58hwb-MuiGrid-root"
                style="display: flex; align-items: center;"
              >
                <h3>
                  Swap
                </h3>
              </div>
            </div>
            <div
              class="MuiGrid-root css-vj1n65-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-1tj2tdk-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-7 css-1smoqvn-MuiGrid-root"
                >
                  <input
                    class="undefined qty-input"
                    placeholder="1000"
                    type="number"
                    value=""
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-14m1l2r-MuiGrid-root"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium undefined coin-select css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <img
                      src="https://github.com/dapphub/trustwallet-assets/blob/master/blockchains/ethereum/info/logo.png?raw=true"
                    />
                    ETH
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth css-9mgopn-MuiDivider-root"
                style="border-color: #202020; margin: 0.5em -1em 0px -1em;"
              />
              <button
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium undefined exchange css-78trlr-MuiButtonBase-root-MuiIconButton-root"
                tabindex="0"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="SettingsEthernetIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7.77 6.76 6.23 5.48.82 12l5.41 6.52 1.54-1.28L3.42 12l4.35-5.24zM7 13h2v-2H7v2zm10-2h-2v2h2v-2zm-6 2h2v-2h-2v2zm6.77-7.52-1.54 1.28L20.58 12l-4.35 5.24 1.54 1.28L23.18 12l-5.41-6.52z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <div
                class="MuiGrid-root MuiGrid-container mt-m1 mb-0 css-1tj2tdk-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-direction-xs-column MuiGrid-grid-xs-7 css-1smoqvn-MuiGrid-root"
                >
                  <input
                    class="undefined qty-input"
                    placeholder="1000"
                    type="number"
                    value=""
                  />
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-5 css-14m1l2r-MuiGrid-root"
                >
                  <button
                    class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium undefined coin-select css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <img
                      src="https://github.com/dapphub/trustwallet-assets/blob/master/blockchains/bitcoin/info/logo.png?raw=true"
                    />
                    BTC
                    <span
                      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                    />
                  </button>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root css-e54n9e-MuiGrid-root"
            >
              <button
                circular="true"
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium Mui-disabled MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium w-100 h-3 gradient-btn-disabled css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                disabled=""
                secondary="true"
                tabindex="-1"
                type="button"
              >
                Enter Amounts to Swap
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      style="display: block;"
    >
      <div
        class="MuiStack-root css-1rzuxdl-MuiStack-root"
      >
        <div
          class="MuiStack-root css-m69qwo-MuiStack-root"
        >
          <div
            class="MuiStack-root css-313u06-MuiStack-root"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 undefined undefined css-ahj2mt-MuiTypography-root"
            >
              Activity
            </p>
          </div>
          <button
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-78trlr-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="KeyboardDoubleArrowRightIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M6.41 6 5 7.41 9.58 12 5 16.59 6.41 18l6-6z"
              />
              <path
                d="m13 6-1.41 1.41L16.17 12l-4.58 4.59L13 18l6-6z"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiStack-root css-nen11g-MuiStack-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-1603n5b-MuiTypography-root"
          >
            No Activities
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
